xcode/.swiftlint.yml

```diff
@@ -227,6 +227,12 @@ custom_rules:
     message: "Returning a boolean as true is redundant, and `!`-syntax is preferred over returning as false."
     severity: error
 
+  single_line_closure:
+    name: "Single line closure"
+    regex: '\{([^\n]*\[[^\]]+\][^\n]*)?([^\n]*[a-zA-Z]+(, (_|[a-zA-Z]+))*)? in [^\n]+'
+    message: "Make a new line after `in` keyword in closure"
```

Comment:
Too complex expression for single line closure. Improve readability by making it multiline (or just "Make it multiline"))